Mathlib/Algebra/Polynomial/RingDivision.lean

```diff
@@ -345,6 +345,15 @@ theorem Monic.C_dvd_iff_isUnit {p : R[X]} (hp : Monic p) {a : R} :
       hp.coeff_natDegree ▸ (C_dvd_iff_dvd_coeff _ _).mp h p.natDegree,
    fun ha => (ha.map C).dvd⟩
 
+theorem natDegree_pos_of_monic_of_ne_one {a : R[X]} (hu : a ≠ 1) (ha : Monic a) :
+    0 < natDegree a := by
+  by_contra! ha'
+  exact hu <| eq_one_of_monic_natDegree_zero ha (by omega)
+
+theorem degree_pos_of_monic_of_ne_one {a : R[X]} (hu : a ≠ 1) (ha : Monic a) :
```

Comment:
```suggestion
theorem Monic.degree_pos {a : R[X]} (hu : a ≠ 1) (ha : Monic a) :
```
or
```suggestion
theorem Monic.degree_pos_of_ne_one {a : R[X]} (hu : a ≠ 1) (ha : Monic a) :
```
or maybe even change the statement to
```
theorem Monic.degree_pos {a : R[X]} (hu : a ≠ 1) (ha : Monic a) :
```

Reply:
Whoop sorry I wrote nonsense at the end of my message. I meant maybe even change the statement to
```
theorem Monic.degree_pos {a : R[X]} (ha : Monic a) : 0 < degree a ↔  a ≠ 1
```